src/ripple/server/impl/BaseWSPeer.h

```diff
@@ -328,7 +388,6 @@ do_read()
     impl().ws_.async_read(op_, rb_, strand_.wrap(
         std::bind(&BaseWSPeer::on_read,
             impl().shared_from_this(), placeholders::error)));
-    cancel_timer();
```

Comment:
We're now sending pings every 30 (or 3) seconds regardless of client activity. Is that what we want?